singolo/index.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>SINGOLO</title>
+  </head>
+  <body>
+    <header class="header" id="header">
+      <div class="header-content">
+        <div class="header__logo">
```

Comment:
Why do you include the burger menu into the logo block?

Reply:
made it separately, so it's better